### PR TITLE
blog: add pos release announcement

### PR DESCRIPTION
**Pull request by RafaelGSS**

### CODE REVIEW

File: pages/en/blog/release/v20.8.1.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+* \[[`f5c90b2951`](https://github.com/nodejs/node/commit/f5c90b2951)] - **module**: fix code injection through export names (Tobias Nießen) [nodejs-private/node-private#461](https://github.com/nodejs-private/node-private/pull/461)
+* \[[`fa5dae1944`](https://github.com/nodejs/node/commit/fa5dae1944)] - **permission**: fix Uint8Array path traversal (Tobias Nießen) [nodejs-private/node-private#456](https://github.com/nodejs-private/node-private/pull/456)
+* \[[`cd35275111`](https://github.com/nodejs/node/commit/cd35275111)] - **permission**: improve path traversal protection (Tobias Nießen) [nodejs-private/node-private#456](https://github.com/nodejs-private/node-private/pull/456)
+* \[[`a4cb7fc7c0`](https://github.com/nodejs/node/commit/a4cb7fc7c0)] - **policy**: use tamper-proof integrity check function (Tobias Nießen) [nodejs-private/node-private#462](https://github.com/nodejs-private/node-private/pull/462)
+
+Windows 32-bit Installer: https://nodejs.org/dist/v20.8.1/node-v20.8.1-x86.msi \
+Windows 64-bit Installer: *Coming soon* \
```
Comment (RafaelGSS) [inline pages/en/blog/release/v20.8.1.md:33]: Is it expected? @richardlau

File: pages/en/blog/release/v18.18.2.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+author: Rafael Gonzaga
+---
+
+### Notable Changes
+
+The following CVEs are fixed in this release:
+
+* [CVE-2023-44487](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-44487): `nghttp2` Security Release (High)
+* [CVE-2023-45143](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-45143): `undici` Security Release (High)
+* [CVE-2023-38552](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38552):  Integrity checks according to policies can be circumvented (Medium)
```
Comment (tniessen) [inline pages/en/blog/release/v18.18.2.md:15]:
```suggestion
* [CVE-2023-38552](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38552): Integrity checks according to policies can be circumvented (Medium)
```

File: pages/en/blog/release/v20.8.1.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+### Notable Changes
+
+The following CVEs are fixed in this release:
+
+* [CVE-2023-44487](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-44487): `nghttp2` Security Release (High)
+* [CVE-2023-45143](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-45143): `undici` Security Release (High)
+* [CVE-2023-39332](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-39332): Path traversal through path stored in Uint8Array (High)
+* [CVE-2023-39331](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-39331): Permission model improperly protects against path traversal (High)
+* [CVE-2023-38552](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38552):  Integrity checks according to policies can be circumvented (Medium)
```
Comment (tniessen) [inline pages/en/blog/release/v20.8.1.md:17]:
```suggestion
* [CVE-2023-38552](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38552): Integrity checks according to policies can be circumvented (Medium)
```

File: pages/en/blog/vulnerability/october-2023-security-releases.md [hunk 4]
```diff
@@ -1,20 +1,112 @@
...
+
+Undici did not always clear Cookie headers on cross-origin redirects. By design, cookie headers are [forbidden request headers](https://fetch.spec.whatwg.org/#forbidden-request-header), disallowing them to be set in RequestInit.headers in browser environments. Since undici handles headers more liberally than the spec, there was a disconnect from the assumptions the spec made, and undici's implementation of fetch.
+
+As such this may lead to accidental leakage of cookie to a 3rd-party site or a malicious attacker who can control the redirection target (ie. an open redirector) to leak the cookie to the 3rd party site.
+
+More details area available in [GHSA-wqq4-5wpv-mx2g](https://github.com/nodejs/undici/security/advisories/GHSA-wqq4-5wpv-mx2g)
+
+## nghttp2 - HTTP/2 Rapid Reset (High) - (CVE-2023-44487)
+
+Rapidly creating and cancelling streams (HEADERS frame immediately followed by RST_STREAM) without bound cause denial of service.
```
Comment (tniessen) [inline pages/en/blog/vulnerability/october-2023-security-releases.md:25]:
```suggestion
Rapidly creating and cancelling streams (`HEADERS` frame immediately followed by `RST_STREAM`) without bound causes denial of service.
```

File: pages/en/blog/vulnerability/october-2023-security-releases.md [hunk 5]
```diff
@@ -1,20 +1,112 @@
...
+See https://www.cve.org/CVERecord?id=CVE-2023-44487 for details.
+
+Impacts:
+
+* This vulnerability affects all users of HTTP2 servers in all active
```
Comment (tniessen) [inline pages/en/blog/vulnerability/october-2023-security-releases.md:30]:
```suggestion
* This vulnerability affects all users of HTTP/2 servers in all active
```

File: pages/en/blog/vulnerability/october-2023-security-releases.md [hunk 6]
```diff
@@ -1,20 +1,112 @@
...
+Impacts:
+
+* This vulnerability affects all users using the experimental policy mechanism in all active release lines: 18.x and, 20.x.
+
+Please note that at the time this CVE is issued, the policy mechanism is an experimental feature of Node.js.
+
+Thanks to [Tobias Nießen](https://github.com/tniessen) who reported and created
+the security patch.
+
+
```
Comment (tniessen) [inline pages/en/blog/vulnerability/october-2023-security-releases.md:82]:
```suggestion

```

File: pages/en/blog/vulnerability/october-2023-security-releases.md [hunk 7]
```diff
@@ -1,20 +1,112 @@
...
 ## Summary
 
 The Node.js project will release new versions of the 18.x and 20.x
 releases lines on or shortly after, Friday October 13 2023 in order to address:
 
 - 2 high severity issues.
-- 1 medium severity issue.
-- 1 low severity issue.
+- 1 medium severity issues.
+- 1 low severity issues.
```
Comment (tniessen) [inline pages/en/blog/vulnerability/october-2023-security-releases.md:109]:
```suggestion
- 1 medium severity issue.
- 1 low severity issue.
```

File: pages/en/blog/vulnerability/october-2023-security-releases.md [hunk 8]
```diff
@@ -32,6 +124,6 @@ Releases will be available on, or shortly after, Friday October 13 2023.
 
 ## Contact and future updates
 
-The current Node.js security policy can be found at <https://nodejs.org/en/security/>. Please follow the process outlined in <https://github.com/nodejs/node/blob/main/SECURITY.md> if you wish to report a vulnerability in Node.js.
+The current Node.js security policy can be found at <https://nodejs.org/en/security/>. Please follow the process outlined in <https://github.com/nodejs/node/blob/master/SECURITY.md> if you wish to report a vulnerability in Node.js.
```
Comment (tniessen) [inline pages/en/blog/vulnerability/october-2023-security-releases.md:127]:
```suggestion
The current Node.js security policy can be found at <https://nodejs.org/en/security/>. Please follow the process outlined in <https://github.com/nodejs/node/blob/main/SECURITY.md> if you wish to report a vulnerability in Node.js.
```

File: pages/en/blog/release/v20.8.1.md (hunk 1 above)
Comment (richardlau) [reply pages/en/blog/release/v20.8.1.md:33]: No. 😕 
```suggestion
Windows 64-bit Installer: https://nodejs.org/dist/v20.8.1/node-v20.8.1-x64.msi \
```

**Review by tniessen — COMMENTED** [review]
Nit: typo `pos` in commit message

**Comment by ovflowd** [comment]
@RafaelGSS did you wait for CI? I don't see that this PR got any approval, nor the CI did pass. Please avoid direct merges when possible 🙇

**Comment by MattIPv4** [comment]
@tniessen or @RafaelGSS would you be able to fast-follow with those suggested changes in a new PR?

**Comment by RafaelGSS** [comment]
I didn't wait for CI, sorry. The release is out and we should have this post asap. I'll create a PR with the suggested changes.

**Comment by tniessen** [comment]
@MattIPv4 I'll open a separate PR. @RafaelGSS probably still has work to do due to the security releases, which take priority :)

**Comment by ovflowd** [comment]
> I didn't wait for CI, sorry. The release is out and we should have this post asap. I'll create a PR with the suggested changes.

Right, the CI takes usually just around 50s-1 minute and half, we just want to ensure that the build passes and works as expected :) (I don't think one extra minute would hurt 🙇)

**Comment by ovflowd** [comment]
Well, `main` is broken now.

**Comment by RafaelGSS** [comment]
Working on that.

**Comment by tniessen** [comment]
Linter errors should be fixed https://github.com/nodejs/nodejs.org/pull/5994.

On a side note, I think the Markdown rules in this repo differ from the Markdown rules in the core repo, but we use some of the same content for both reps.

**Comment by ovflowd** [comment]
> On a side note, I think the Markdown rules in this repo differ from the Markdown rules in the core repo, but we use some of the same content for both reps.

Are you referring to the `*` instead of `-`? We don't use the `remark-preset-lint-node` as it is incompatible with MDX and causes serious issues.

But we can change the rules to use `*` instead of `-` as it is on Node core. Besides of that, I would always recommend to:

1. Run your changes locally
2. Ensure you have the Git Hooks installed (simply do `npm i`
3. Ensure it builds locally
4. Commit (linting is done automatically)
5. CI on a PR.

**Comment by tniessen** [comment]
> Are you referring to the `*` instead of `-`?

That's the main difference that I noticed so far :)

**Comment by ovflowd** [comment]
> > Are you referring to the `*` instead of `-`?
> 
> That's the main difference that I noticed so far :)

Apparently prettier enforces `-`, there's nothing I can do here, at least I can try to make the ride smoother... https://github.com/nodejs/nodejs.org/pull/5997

**Pull request merged**